packages/wallet/backend/src/auth/controller.ts

```diff
@@ -63,4 +65,22 @@ export class AuthController implements IAuthController {
       next(e)
     }
   }
+
+  logOut = async (req: Request, res: CustomResponse, next: NextFunction) => {
+    try {
+      if (!req.session.id || !req.session.user) {
+        req.session.destroy()
+        throw new Unauthorized('Unauthorized')
+      }
+
+      await this.deps.authService.logout(req.session.user.id)
+      req.session.destroy()
+      res.clearCookie(env.COOKIE_NAME)
```

Comment:
`req.session.destroy()` automatically clears the cookie.
https://github.com/vvo/iron-session#sessiondestroy